Content.Server/GameObjects/Components/Chemistry/VaporHelper.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Content.Shared.Chemistry;
+using Robust.Server.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.Map;
+using Robust.Shared.IoC;
+using Robust.Shared.Map;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Content.Server.GameObjects.Components.Chemistry
+{
+    static class VaporHelper // Basically the SpillHelper but for vapor entities
+    {
+        /// <summary>
+        /// Spawns the specified solution as a vapor at the entity's location if possible.
+        /// </summary>
+        /// <param name="entity">Entity to spawn the vapor at</param>
+        /// <param name="solution">Initial solution for the prototype</param>
+        internal static void SpillAt(IEntity entity, Solution solution)
+        {
+            var entityLocation = entity.Transform.GridPosition;
+            SpillAt(entityLocation, solution);
```

Comment:
Return VaporComponent here, perhaps?

Reply:
Literally stole that from SpillHelper. Should I add it there too?